network-upgrades/README.md

```diff
@@ -0,0 +1,42 @@
+# Network Upgrades Specifications
+
+## Description
+
+This folder provides the specifications for the various network upgrades to Ethereum. It contains specifications for the client integration testnets which are set up during the testing phase of upgrades, for the network upgrades which will be deployed to the Ethereum mainnet, as well as retrospectives/postmortems on network upgrade-related incidents.
+
+In an effort to separate this process from the EIP process itself, three stages have been devised to move EIPs from specifications to being deployed on mainnet: Considered for Inclusion (CFI), Client Integration Testnets, and Mainnet. Below, we explain the process for EIPs to move through these stages.
+
+## Definitions
+
+**Considered for Inclusion**: Signals that client developers are generally positive towards the idea, and that, assuming it meets all the requirements for mainnet inclusion, it could potentially be included in a network upgrade. This means the EIP may be included in client integration testnets. It is similar to "concept ACK" in other open source projects, and is not sufficient to result in deployement to mainnet.
```

Comment:
```suggestion
**Considered For Inclusion**: CFI approved signals that client developers are generally positive towards the idea, and that, assuming it meets all the requirements for the mainnet inclusion, it could potentially be included in a network upgrade. This means the EIP may be included in the client integration testnets. It is similar to "concept ACK" in other open-source projects, and is not sufficient to result in deployment to the mainnet.
```

Reply:
I'm not sure we should call it "CFI approved": we can just say that when something is considered for inclusin, then it signals that client devs like it. 

Also don't think all the "the" are needed.

Reply:
Just suggestions, fine with the original texts.